src/Error.php

```diff
@@ -51,14 +51,21 @@ public function getContainerFor( DIProperty $property = null, $errorMsg = '' ) {
 			$property = new DIProperty( $property->getKey() );
 		}
 
-		$subWikiPage = new DIWikiPage(
+		$errorMsg = is_array( $errorMsg ) ? implode( ' ', $errorMsg ) : $errorMsg;
+
+		$subject = new DIWikiPage(
 			$this->subject->getDBkey(),
 			$this->subject->getNamespace(),
 			$this->subject->getInterwiki(),
-			'_ERR' . md5( $property !== null ? $property->getKey() : 'UNKNOWN' )
+			'_ERR' . md5( ( $property !== null ? $property->getKey() : 'UNKNOWN' ) . $errorMsg )
 		);
 
-		$containerSemanticData = new ContainerSemanticData( $subWikiPage );
+		// Encode brackets to avoid an annotion is created/included
+		return $this->newDiContainer( $subject, $property, str_replace( '[', '&#x005B;', $errorMsg ) );
+	}
```

Comment:
@kghbln This fixes [0] where a queried error list copied `[[ ... ]]` annotations from the query result into a page.

[0] https://wikiapiary.com/wiki/Special:Browse/User:Thingles-2FFailed_queries